dataset.html

```diff
@@ -130,11 +130,14 @@ <h2 id="readme">README</h2>
 
   </div><!-- / dataset show -->
 </div>
-
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
```

Comment:
Why is this in the page? I would imagine we want this in the built bundle.js (i.e. it should be a dependency).

Also do we need this because handsontable depends on jquery?